customize_image.sh

```diff
@@ -314,27 +312,19 @@ if [ -e "$IMAGEDIR/etc/init.d/apache2" ] ; then
 fi
 
 if [ "$MINIMG" != "1" ]; then
-	# fake a revpi procfs ... otherwise NodeRed installer will get confused
```

Comment:
Please use git revert for 79d307c58bb3cf951141c62b5ae56fe71d7875f5, this will allow us to remove all reverted commits from the bullseye branch before we merge it to master.